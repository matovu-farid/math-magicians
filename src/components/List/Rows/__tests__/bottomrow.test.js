import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import BottomRow from '../bottomrow';

const mockHandleButtonClick = jest.fn();
it('renders correctly', () => {
  const tree = renderer.create(<BottomRow handleButtonClick={mockHandleButtonClick} />).toJSON();
  expect(tree).toMatchSnapshot();
});
it('should have 2 buttons', () => {
  render(<BottomRow handleButtonClick={mockHandleButtonClick} />);
  const listItems = screen.queryAllByRole('listitem');

  expect(listItems.length).toBe(2);
});
