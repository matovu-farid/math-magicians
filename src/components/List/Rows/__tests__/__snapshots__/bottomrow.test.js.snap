// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul
  className="top-list"
>
  <li>
    <button
      className="tile  wide"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
  </li>
  <li>
    <button
      className="tile"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
  </li>
</ul>
`;
